xdsl/dialects/snitch_runtime.py

```diff
@@ -46,9 +46,9 @@ def __init__(
         super().__init__(operands=[], result_types=[i32])
 
 
-class SnitchRuntimeBarrier(SnitchRuntimeBaseOp, ABC):
+class NoOperandNoResultBaseOp(SnitchRuntimeBaseOp, ABC):
```

Comment:
```suggestion
class NoOperandNoResultBaseOperation(SnitchRuntimeBaseOp, ABC):
```
This isn't really an official rule, more of a convention, but so far the abstract Op classes have Operation in the name, and the concrete ones have Op, which also kind of reflects that the Operation ones are in the code less, seems like it would be good to be consistent there